webpack/entry.js

```diff
@@ -1,6 +1,17 @@
-window.$ = require('jquery');
-window.Mustache = require('mustache');
+window.$ = require("jquery");
+window.Mustache = require("mustache");
+import Vue from "vue";
+import "bootstrap-sass/assets/javascripts/bootstrap.js";
 
-import 'bootstrap-sass/assets/javascripts/bootstrap.js';
+import "./scss_files/main.scss";
 
-import './scss_files/main.scss';
\ No newline at end of file
+import MeetupEvents from "../js/vue/meetupEvents.vue";
+
+document.addEventListener("DOMContentLoaded", function() {
+  var eventsApp = new Vue({
+    el: "#meetup-events",
+    render: function(h) {
+      return h(MeetupEvents);
+    }
+  });
```

Comment:
This will crash on other pages that don't have a `#meetup-events`. To make this a little more flexible I'd like to propose the following change:

```patch
﻿From e4a5c5f12cbf1bf034b91ee7f93586f3d9773915 Mon Sep 17 00:00:00 2001
From: Devin Weaver <suki@tritarget.org>
Date: Thu, 10 May 2018 19:24:15 -0400
Subject: [PATCH] Allow multiple islands of richness

Instead of presuming the *every* page will include a meetup-events div
we can add them conditionally. This gives us the ability to opt-in to
islands of richness on a page by adding a div with a known ID.
---
 webpack/entry.js | 16 +++++++++++-----
 1 file changed, 11 insertions(+), 5 deletions(-)

diff --git a/webpack/entry.js b/webpack/entry.js
index a7b58e6..c5a69e3 100644
--- a/webpack/entry.js
+++ b/webpack/entry.js
@@ -8,10 +8,16 @@ import "./scss_files/main.scss";
 import MeetupEvents from "../js/vue/meetupEvents.vue";
 
 document.addEventListener("DOMContentLoaded", function() {
-  var eventsApp = new Vue({
-    el: "#meetup-events",
-    render: function(h) {
-      return h(MeetupEvents);
-    }
+  var islandsOfRichness = {
+    "meetup-events": MeetupEvents
+  };
+
+  Object.keys(islandsOfRichness).forEach(function(tagId) {
+    var appElement = document.getElementById(tagId);
+    if (!appElement) { return; }
+    new Vue({
+      el: appElement,
+      render: function(h) { return h(islandsOfRichness[tagId]); }
+    });
   });
 });
-- 
2.17.0
```